tests/unitutil.py

```diff
@@ -2,9 +2,11 @@
 
 """Functions that make mocking with pytest easier and more readable."""
 
+from unittest.mock import (ANY, PropertyMock, call, create_autospec,  # noqa
```

Comment:
There was a reason to have that 2 import statement broken in two lines. The `# noqa` comment is used to skip the coverage on ANY and call, but we need to check the coverage on the property mock and patch and so on... so i would leave that two lines as before